corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -583,6 +583,31 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
             </div>
             </div>
           {% endif %}
+	  {% if request|toggle_enabled:"RESTRICT_MOBILE_ACCESS" %}
```

Comment:
Nitpick: is this file mixing tabs and spaces? Indentation is weird.

Reply:
Ugh, yeah, that's my bad. Thanks for catching. Will clean up before I push